bindings/java/iota-wallet-java/examples/src/main/Env.java

```diff
@@ -0,0 +1,11 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+public class Env {
+    public static final String NODE = "https://api.testnet.shimmer.network";
+    public static final String EXPLORER = "explorer.shimmer.network/testnet";
+    public static final String PASSWORD = "ENCRYPTION_PASSWORD";
+    public static final String ACCOUNT_NAME = "Alice";
+    public static final String SNAPSHOT_PATH = "example-wallet";
```

Comment:
maybe rename to `STRONGHOLD_SNAPSHOT_PATH` to make it more clear what kind of snapshot this is.
btw. should we call it even snapshot? or should it be better called like "STRONGHOLD_STORE" or "STRONGHOLD_VAULT" or similar?

Reply:
@Thoralf-M any opinion on this?

Reply:
In stronghold it's called snapshot, so I think keeping `SNAPSHOT` is fine, but yeah we can preprend it with `STRONGHOLD`